### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Integralist**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,11 +20,12 @@ Speed up your websites with a Compute@Edge environment that demonstrates serving
 
 ## Usage
 
-When initializing your project, the Fastly CLI will prompt you to enter a `Backend`. You can enter your bucket host here, or just enter `mock-s3.edgecompute.app` on port `443` if you want to experiment with our mock backend.
+When deploying your project, the Fastly CLI (`v1.1.0`) will prompt you to enter a `Backend`. You can enter your bucket host here, or just enter `mock-s3.edgecompute.app` on port `443` if you want to experiment with our mock backend.
```
Comment (Integralist) [inline README.md:23]: @triblondon let me know if specifying the version of the CLI in this way is OK.

Comment (Integralist) [reply README.md:23]: @triblondon In a separate repo you suggested we get into the habit of updating CLI instructions after each new release, so I presume the same principle applies to the starter kits.

**Comment by Integralist** [comment]
As noted here:
https://blog.rust-lang.org/inside-rust/2020/01/23/Introducing-cargo-audit-fix-and-more.html

I needed to install a special build of `cargo audit` to get it to auto-fix the issues for me...

```
cargo install cargo-audit --features=fix
```

Now I can run:

```
cargo audit fix
```

**Comment by Integralist** [comment]
LOL. Seems `cargo audit fix` doesn't _actually_ fix the issue. Maybe it's because it's a nested dependency and as the fix subcommand is still 'experimental' it's not ready to work with deeply nested dependency resolution?

**Comment by Integralist** [comment]
So the `cargo audit` messaging suggests there is no safe version of `Chrono` dependency to switch to. 

Reading through https://github.com/chronotope/chrono/issues/602 it hints at Chrono being _unmaintained_ and so some people are switching back to a more recent release of the `time` dependency (which I believe Chrono was supposed to be a superset of).

@jedisct1 are you able to advise on the best action to take here with fixing the vulnerabilities reported?

**Comment by jedisct1** [comment]
Ah, scratch that, the `x-amz-date` header actually requires to format the date.

Until a long-term solution is found, it is safe to ignore the warning, by adding `--ignore RUSTSEC-2020-0159` to the `cargo audit` command. The soundness issue in the `time` crate `chrono` depends on doesn't affect the way we use it.

**Comment by jedisct1** [comment]
There's also a minimal alternative: https://docs.rs/time-format

**Review by jedisct1 — APPROVED** [review]

**Pull request merged**